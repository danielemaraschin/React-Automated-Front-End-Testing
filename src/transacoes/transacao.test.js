import React from 'react';
import { render } from '@testing-library/react';
import Transacao from './Transacao';

describe('Componente de Transação no extrato', () => {
    
        it('O snapshot do componente deve ser sempre o mesmo ', () => {
            const { container} = render(<Transacao //FAZ DESTRUCTURING  DO RENDER
                data="08/09/2020"
                tipo="saque"
                valor="20.00"
            />);

            expect(container.firstChild).toMatchSnapshot();
        })
});
//sempre c    
//container é uma const que o render nos dá
 //o firstchild do container será o elemento HTML já montado  
//ele tem outros elelmentos tb, mas nao usaremos agora
//comitar o arquivo de snapshots para que o servidor consiga fazer a comparação
