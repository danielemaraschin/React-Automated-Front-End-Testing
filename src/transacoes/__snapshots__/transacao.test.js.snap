// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de Transação no extrarto  o snapshot do componente deve ser sempre o mesmo  1`] = `undefined`;
 

